server/middleware/middleware.js

```diff
@@ -0,0 +1,215 @@
+import validator from 'validator';
+import jwt from 'jsonwebtoken';
+
+require('dotenv').config();
+
+
+/** middleware class */
+class middleware {
+  /**
+   *
+   * @param {object} req
+   * @param {object} res
+   * @param {object} next
+   * @returns {object} next
+   */
+  static authenicateAdmin(req, res, next) {
+    const token = req.headers['x-access-token'] || req.body.token || req.query.token;
+    jwt.verify(token, process.env.SECRET, (err, decoded) => {
+      if (err) {
+        return res.status(403).json({ success: false, message: 'Forbidden to non admin' });
```

Comment:
This error does not check for the admin role why does message say `Forbidden to non admin`?